app/views/welcome/index.html.erb

```diff
@@ -4,6 +4,13 @@
 <div class="col-md-6 col-md-offset-3 text-center">
     <h3>Future Home Of the Fall Gambit Chess App!</h3>
     <br />
-    <%= render "available_games" %>
+    <% if @games.present? %>
+      <%= render partial: 'available_games' %>
+    <% else %>
+      <h2>
+        No games available...
+      </h2>
+      <%= link_to 'Create Game', new_game_path, class: 'btn btn-success btn-large' %>
+    <% end %>
```

Comment:
Also shouldn't we have a link to the new_game_path regardless of the available games? What if I want to play a game against a specific opponent and that game isn't currently extant? How do I create the new game?